SharedPackages/BrowserServicesKit/Sources/PixelKit/PixelKitEvent.swift

```diff
@@ -19,9 +19,14 @@
 import Foundation
 import Common
 
+public enum PixelKitStandardParameter {
+    case pixelSource
```

Comment:
We could add `appVersion` here and remove it from the parameters list too, but I avoided it for now since the PR is very large. Happy to make the change for completeness though.